CONTRIBUTING.md

```diff
@@ -350,6 +353,38 @@ This involves the following steps:
    ```bash
    >>> git push origin --delete feature-add-new-important-headers
    ```
+## Testing the documentation
+
+`oomph-lib`'s extensive documentation is generated by doxygen and the full set of webpages (containing many detailed tutorials) is hosted on Github. The webpages are regenerated automatically whenever the `main` branch of the official `oomph-lib` repository is updated. It is therefore important that you check any changes to the documentation and make sure that they render correctly before they are ultimately committed to the main branch. It is therefore important that you check any changes to the documentation and make sure that they render correctly before they are ultimately committed to the `main` branch.
```

Comment:
- Up to your discretion but I think "doxygen" could/should be replaced with either Doxygen, i.e. the name of the official tool, or `doxygen`, the name of the command-line tool. 
- Need to delete the first "It is therefore important that you check any changes to the documentation and make sure that they render correctly before they are ultimately committed to the main branch."  (it's duplicated)

Reply:
Done (also added a link to doxygen page and removed the duplicate (brain fog; I'll send you link that'll explain why).